cart.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>FlameKeys | Cart Page </title>
+    <script src="https://kit.fontawesome.com/d3026d29a6.js" crossorigin="anonymous"></script>
+
+    <link rel="stylesheet" href="./main.css">
+    <link rel="stylesheet" href="https://flameui.netlify.app/components.css">
+    <link rel="stylesheet" href="./cart.css">
+</head>
+
+<body>
+    <div class="cart-page-container">
+        <!-- Nav bar starts here -->
+        <div class="nav-bar">
+            <a class="homepage-nav-links" href="./index.html" target="_blank">
+                <div class="homepage-nav-logo">
+                    <div>FlameKeys</div>
+                    <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
```

Comment:
use semantics tags

Reply:
This change will require me to make changes in html code of multiple files, so will make the changes later, but thanks for your suggestion :)